### PR TITLE
homepage6

### PR DESCRIPTION
**Pull request by EYRAM565**
homepage
Screenshot (41)
## Attach Screenshot

> Note 2 code reviewer approval needed. Approach in twitter group & discord channel.

### CODE REVIEW

**Comment by shekhar10feb** [comment]
Please remove last  </div>

**Comment by EYRAM565** [comment]
last? can you be specific

**Comment by shekhar10feb** [comment]
> 
> 
> last? can you be specific

yes

**Comment by shekhar10feb** [comment]
means last </div>

**Review by shekhar10feb — APPROVED** [review]
Looking nice!!

File: playground/twindleco/homepage/homapage6/home.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
+:root{
+    background-color: 	#fef6e4;
```
Comment (ni3-kdm) [inline playground/twindleco/homepage/homapage6/home.css:2]: @EYRAM565 follow code formatting rules

File: playground/twindleco/homepage/homapage6/home.css [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    height: 70px;  
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;   
+    margin-left: 50px; 
+    color: #001858;
+
+}
+
+.link a{
```
Comment (ni3-kdm) [inline playground/twindleco/homepage/homapage6/home.css:27]: @EYRAM565 follow code formatting rules

**Comment by ni3-kdm** [comment]
@EYRAM565 Refer [CSS style guide rules](https://gist.github.com/fleishmanhillard/9983654). Must go through each one and use accordingly.

**Comment by EYRAM565** [comment]
okay.

**Comment by proful** [comment]
@ni3-kdm thanks for the suggestion. Following a style guide definitely helps.

However merging this pr. Those changes can be raised as separate pr. As the code is going to playground.

**Pull request merged**